test/fixtures/autofix/jQuerySap.js

```diff
@@ -0,0 +1,237 @@
+sap.ui.define(["sap/base/strings/NormalizePolyfill"], function (NormalizePolyfill) {
+	// https://github.com/SAP/ui5-linter/issues/520
+	jQuery.sap.assert(false, "That's an assert");
+
+	// https://github.com/SAP/ui5-linter/issues/521
+	const bundle = jQuery.sap.resources({
+		url: "resources/i18n.properties"
+	});
+	// Respect the context
+	sap.ui.require(["sap/base/i18n/ResourceBundle"], function (innerScopeResourceBundle) {
+		const bundle = jQuery.sap.resources({
+			url: "resources/i18n.properties"
+		});
+	});
+
+	// https://github.com/SAP/ui5-linter/issues/522
+	const logObject = jQuery.sap.log;
+	const myLogger = jQuery.sap.log.getLogger();
+	const myLog = jQuery.sap.log.getLog();
+	const myLog2 = jQuery.sap.log.getLogEntries();
+
+	const oLogListener = {};
+	oLogListener.onLogEntry = function(oLog) {
+		//
+	};
+	jQuery.sap.log.addLogListener(oLogListener);
+	jQuery.sap.log.removeLogListener(oLogListener);
+
+	const level = jQuery.sap.log.getLevel();
+	const logLevel = jQuery.sap.log.Level;
+	const isLoggable = jQuery.sap.log.isLoggable(logLevel.DEBUG);
+
+	jQuery.sap.log.logSupportInfo(true);
+	jQuery.sap.log.debug("This is a debug log message");
```

Comment:
The other args are not covered in our tests: `sDetails?, sComponent?, fnSupportInfo?`.

Reply:
Done